src/internal/marked/Renderer.js

```diff
@@ -1,30 +1,35 @@
 import {h} 					from 'preact/preact';
 
-import Util 				from './Util';
+import Util					from './Util';
 
 //COMPONENT IMPORTS
-import NavLink 				from 'com/nav-link/link';
 import LinkMail				from 'com/link-mail/mail';		// TODO: Obsolete me
+import NavLink 				from 'com/nav-link/link';
+import SmartLink 			from 'com/autoembed/smartlink';
+import LocalLink			from 'com/autoembed/locallink';
+
 import AutoEmbed 			from 'com/autoembed/autoembed';
+import {SmartDomains} 		from 'com/autoembed/smartdomains';
```

Comment:
Any reason `{SmartDomains}` is used here, rather than just `SmartDomains`?

Reply:
I changed the syntax in `smartdomains` so that i can remove those braces.